### PR TITLE
Standardize integration tests

### PR DESCRIPTION
**Pull request by ingwarr**
Fixes  https://github.com/terraform-google-modules/terraform-google-memorystore/issues/9 fixes https://github.com/terraform-google-modules/terraform-google-memorystore/issues/5

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -44,3 +44,4 @@ test/fixtures/*/terraform.tfvars
 terraform.tfstate.d/
 *.auto.tfvars
 credentials.json
+/.idea/
```
Comment (paulpalamarchuk) [inline .gitignore]: Not sure that we need it

**Comment by ivankorn** [comment]
@ingwarr, if you wanted to closes the mentioned issues with this pr you need fixes/closes etc word before each issue:

fixes #4 fixes #9 etc

File: .gitignore (hunk 1 above)
Comment (ingwarr) [reply .gitignore]: We mostly use Idea as the editor, so we need it

**Comment by ingwarr** [comment]
image

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,17 @@ project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased]
 ### Changed
 
+### Added
+
+- Add support for Terraform 0.12 [11]
```
Comment (nick4fake) [inline CHANGELOG.md]: Please change to #11

File: variables.tf [hunk 3]
```diff
@@ -16,62 +16,76 @@
...
 variable "display_name" {
   description = "An arbitrary and optional user-provided name for the instance."
+  type        = string
   default     = ""
 }
 
 variable "reserved_ip_range" {
   description = "The CIDR range of internal addresses that are reserved for this instance."
+  type        = string
   default     = ""
```
Comment (nick4fake) [inline variables.tf]: Please check if null is more appropriate

Comment (ingwarr) [reply variables.tf]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (ingwarr) [reply CHANGELOG.md]: Done

**Comment by ingwarr** [comment]
All tests passed after rebase
image

**Comment by morgante** [comment]
Closing because we should simply update to use Cloud Build and the new workflow.

**Pull request closed without merging**